config/formats.js

```diff
@@ -536,83 +536,161 @@ let Formats = [
 		column: 2,
 	},
 	{
-		name: "[Gen 7] 2v2 Doubles",
-		desc: `Double battle where you bring four Pok&eacute;mon to Team Preview and choose only two.`,
+		name: "[Gen 7] Fortemons",
+		desc: `Pok&eacute;mon have all of their moves inherit the properties of the move in their item slot.`,
 		threads: [
-			`&bullet; <a href="https://www.smogon.com/forums/threads/3606989/">2v2 Doubles</a>`,
+			`<a href="https://www.smogon.com/forums/threads/3638520/">Fortemons</a>`,
```

Comment:
I don’t know why I accidentally removed the bullet here but I can’t edit it until late tomorrow so if this is ready to be merged by then then feel free to do it